src/main/java/crazypants/enderio/machine/spawner/BlockPoweredSpawner.java

```diff
@@ -346,32 +344,32 @@ public void addDetailedEntries(ItemStack itemstack, EntityPlayer entityplayer, L
     }
   }
 
-  @Override
-  public void getWailaInfo(List<String> tooltip, EntityPlayer player, World world, int x, int y, int z) {
-    TileEntity tileEntity = world.getTileEntity(new BlockPos(x, y, z));
-    if (tileEntity instanceof TilePoweredSpawner) {
-      CapturedMob capturedMob = ((TilePoweredSpawner) tileEntity).getEntity();
-      if (capturedMob != null) {
-        tooltip.add(capturedMob.getDisplayName());
-      }
-    }
-  }
-
-  @Override
-  public int getDefaultDisplayMask(World world, int x, int y, int z) {
-    return IWailaInfoProvider.BIT_DETAILED;
-  }
+//  @Override
+//  public void getWailaInfo(List<String> tooltip, EntityPlayer player, World world, int x, int y, int z) {
+//    TileEntity tileEntity = world.getTileEntity(new BlockPos(x, y, z));
+//    if (tileEntity instanceof TilePoweredSpawner) {
+//      CapturedMob capturedMob = ((TilePoweredSpawner) tileEntity).getEntity();
+//      if (capturedMob != null) {
+//        tooltip.add(capturedMob.getDisplayName());
+//      }
+//    }
+//  }
+//
+//  @Override
+//  public int getDefaultDisplayMask(World world, int x, int y, int z) {
+//    return IWailaInfoProvider.BIT_DETAILED;
+//  }
 
   @SuppressWarnings("null")
   @Override
   @SideOnly(Side.CLIENT)
-  public void getSubBlocks(Item item, CreativeTabs tab, List<ItemStack> list) {
+  public void getSubBlocks(Item item, CreativeTabs tab, NonNullList<ItemStack> list) {
     super.getSubBlocks(item, tab, list);
-    list.add(CapturedMob.create("Enderman", null).toStack(item, 0, 1));
-    list.add(CapturedMob.create("Chicken", null).toStack(item, 0, 1));
-    list.add(CapturedMob.create("Skeleton", SkeletonType.NORMAL).toStack(item, 0, 1));
-    list.add(CapturedMob.create("Skeleton", SkeletonType.WITHER).toStack(item, 0, 1));
-    list.add(CapturedMob.create("Skeleton", SkeletonType.STRAY).toStack(item, 0, 1));
+//    list.add(CapturedMob.create("Enderman", null).toStack(item, 0, 1));
```

Comment:
Why comment these?

Reply:
Because I was lazy and I didn't want to look into getting the id's for all of them.